src/rcdom.rs

```diff
@@ -45,6 +45,7 @@ pub enum ElementEnum {
     /// A template element and its template contents.
     /// https://html.spec.whatwg.org/multipage/#template-contents
     Template(Handle),
+    AnnotationXml(bool),
```

Comment:
I didn't add comments yet, it was just supposed to be a first shot to get help. on some details before finishing it. I should have precised, my bad.